erpnext/accounts/doctype/accounting_dimension_filter/accounting_dimension_filter.py

```
@@ -8,13 +8,38 @@
+
+	from typing import TYPE_CHECKING
+
+	if TYPE_CHECKING:
+		from frappe.types import DF
+
+		from erpnext.accounts.doctype.allowed_dimension.allowed_dimension import AllowedDimension
+		from erpnext.accounts.doctype.applicable_on_account.applicable_on_account import ApplicableOnAccount
+
+		accounting_dimension: DF.Literal[None]
+		accounts: DF.Table[ApplicableOnAccount]
+		allow_or_restrict: DF.Literal["Allow", "Restrict"]
```

Ruff (0.14.5)

Undefined name `Allow`

(F821)